liquid_node/docker.py

```diff
@@ -10,33 +10,37 @@ def containers(self, labels=[]):
         out = run(f'docker ps -q {label_args}')
         return out.split()
 
-    def exec_command(self, name, *args, tty=False):
+    def exec_command(self, name, *args, interactive=False, tty=False):
         """Prepare and return the command to run in a docker container tagged with
         liquid_task=`name`
-
         :param name: the value of the liquid_task tag
-        :param tty: if true, instruct docker to allocate a pseudo-TTY and keep stdin open
+        :param tty: if true, instruct docker to allocate a pseudo-TTY
+        :param interactive: if true docker will keep STDIN open
         """
 
+        docker_exec_cmd = ['docker', 'exec']
+
         containers = self.containers([('liquid_task', name)])
         container_id = first(containers, f'{name} containers')
 
-        docker_exec_cmd = ['docker', 'exec']
         if tty:
-            docker_exec_cmd += ['-it']
-        docker_exec_cmd += [container_id] + list(args or (['bash'] if tty else []))
-
+            docker_exec_cmd += ['-t']
+        if interactive:
+            docker_exec_cmd += ['-i']
+        docker_exec_cmd += [container_id] + list(args or (['bash'] if (tty and interactive) else []))
```

Comment:
Maybe it's time to drop this `bash` default thing. Docker invokes the default entrypoint anyway.

Reply:
that would break `liquid shell <container>` without any argument, since docker exec needs a command.
```
DEBUG + ['docker', 'exec', '-t', '-i', '723a0878cfde']
"docker exec" requires at least 2 arguments.
See 'docker exec --help'.

Usage:  docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
```
Or did you mean that everybody should just write `liquid shell <container> bash` if they want to bash into the container?